lib/patch-series.ts

```diff
@@ -896,7 +935,7 @@ export class PatchSeries {
         if (subjectPrefix) {
             args.push("--subject-prefix=" + subjectPrefix);
         }
-        if (this.coverLetter) {
+        if (this.coverLetter && patchCount > 1) {
```

Comment:
It seems like the code around line ~913 could be moved here.  Might make the logic a little clearer.
```
if (this.patchCount > 1 ) {
   if (this.coverLetter) {
      args.push("--cover-letter");
   } else {
        throw new Error("Branch " + this.project.branchName
                + " needs a description");
  }
}
```

Reply:
Good idea! I did not find any good commit to squash this into, so I left it as an add-on patch.